### DISCUSSION TITLE
Open in app should open the repo in the mobile app

### DISCUSSION THREAD

**Discussion by sshquack** [community/community#14001] [Mobile] [answered]
Here is how "Open" from a Github repo in the mobile browser -> Github app is broken (Notice that the actual repo is not opened in the mobile app)

https://user-images.githubusercontent.com/5678459/161116058-48fa8222-2abc-4a18-a10d-258234fef5af.mov

 
Here is an example of how this should work. When you browse a reddit page and "Open in app" -> the reddit app opens -> takes you to the same reddit page:

https://user-images.githubusercontent.com/5678459/161116282-08539dca-f1cd-4fce-a510-7169be18416b.mov

**Comment by jacortinas** [comment, marked as answer]
Thank you for more great feedback, @sshquack! I've created an issue to track this feature request. 🙇

**Reply by sshquack** [reply]
Thanks @jacortinas Looking forward to this one too!

It might be related to https://github.com/github/feedback/discussions/14002
